package.json

```diff
@@ -123,15 +123,15 @@
     "website:start-static": "cross-env NODE_OPTIONS=\"--max-old-space-size=6144\" TS_NODE_PROJECT=tsconfig/tsconfig.es2017.cjs.json yarn run react-static start",
```

Comment:
The `website:start-prod` script uses line 122 above this line for the server. Compare to the `website:start-server-prod` script below. The `website:start-server-prod` script doesn't seem to work. Using the `serve` package just makes the compiled files available. Adding the `--watch` flag like this command does works though.